x-pack/plugins/siem/public/components/page/network/users_table/columns.tsx

```diff
@@ -4,25 +4,15 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
-import { StaticIndexPattern } from 'ui/index_patterns';
-
 import { FlowTarget, UsersItem } from '../../../../graphql/types';
-import { networkModel } from '../../../../store';
-import { DragEffects, DraggableWrapper } from '../../../drag_and_drop/draggable_wrapper';
-import { escapeDataProviderId } from '../../../drag_and_drop/helpers';
-import { getEmptyTagValue } from '../../../empty_value';
+import { defaultToEmptyTag } from '../../../empty_value';
 import { Columns } from '../../../load_more_table';
-import { Provider } from '../../../timeline/data_providers/provider';
 
 import * as i18n from './translations';
-import { getRowItemDraggables } from '../../../tables/helpers';
+import { getRowItemDraggables, getRowItemDraggable } from '../../../tables/helpers';
 
 export const getUsersColumns = (
-  indexPattern: StaticIndexPattern,
-  ip: string,
   flowTarget: FlowTarget,
-  type: networkModel.NetworkType,
```

Comment:
Thanks for cleaning up these args! Looks like I had them leftover from previous `AddToKql` support 😬